### PR TITLE
feature/unlock-wallet-by-mnemonic

### PR DESCRIPTION
**Pull request by WrBug**
### Devop  
* [ ] Updated CHANGELOG.md  
* [x] Add PR label

### CODE REVIEW

File: src/layouts/WalletNotFoundLayout/WalletNotFoundLayout.scss [hunk 1]
```diff
@@ -0,0 +1,47 @@
+@import '~@/scss/GlobalVariables';
```
Comment (gamalielhere) [inline src/layouts/WalletNotFoundLayout/WalletNotFoundLayout.scss]: This scss file and the corresponding layout file has been moved inside the interface layout folder. You should rebase or pull down the current version or remove this for duplication.

File: src/App.vue [hunk 2]
```diff
@@ -44,9 +44,9 @@ export default {
 
     this.$store.dispatch('setState', state)
     this.$store.dispatch('checkIfOnline')
-    if (window.web3) {
-      this.$store.dispatch('setWeb3Instance', window.web3)
-    }
+    // if (window.web3) {
```
Comment (gamalielhere) [inline src/App.vue]: This is checking for any instance of web3 in the window. This should be kept. Along with the line 23 change.

File: src/router/index.js [hunk 3]
```diff
@@ -60,6 +61,11 @@ export default new Router({
       path: '*',
       name: '404',
       component: NotFoundLayout
+    },
+    {
+      path: '/wallet-not-found',
+      name: 'WalletNotFoundLayout',
+      component: WalletNotFoundLayout
```
Comment (gamalielhere) [inline src/router/index.js]: Wallet not found should only be shown when the user goes to /interface without an unlocked wallet. Please remove.

File: src/layouts/WalletNotFoundLayout/index.js [hunk 4]
```diff
@@ -0,0 +1 @@
+export { default } from './WalletNotFoundLayout'
```
Comment (gamalielhere) [inline src/layouts/WalletNotFoundLayout/index.js]: See comment on `WalletNotFound.scss`.

File: src/layouts/WalletNotFoundLayout/WalletNotFoundLayout.vue [hunk 5]
```diff
@@ -0,0 +1,38 @@
+<template>
```
Comment (gamalielhere) [inline src/layouts/WalletNotFoundLayout/WalletNotFoundLayout.vue]: See comment on `WalletNotFound.scss`.

File: src/App.vue (hunk 2 above)
Comment (WrBug) [reply src/App.vue]: image

I try to print web3's version , result is 0.20.3 . ,   but the dependent version is  1.0.0-beta.34  , what is the reason?

**Pull request closed without merging**